modules/command/dc/dc-write-to-db-command.js

```diff
@@ -0,0 +1,136 @@
+const Command = require('../command');
+const { forEachSeries } = require('p-iteration');
+const Utilities = require('../../Utilities');
+const { sha3_256 } = require('js-sha3');
+
+class DcWriteToDbCommand extends Command {
```

Comment:
Could you name this command: `DCWriteImportToGraphDbCommand`? 
The `DCOfferCreateDbCommand` writes to SQL database, so it might be useful to make the distinction and keep some kind of similarity to the naming. Write to db is not obvious that it's referring to an import.